src/store/rockets/rockets.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+
+const GET_ROCKETS = 'space-trevelers-hub/rockets/GET_ROCKETS';
+const SET_ROCKETS = 'space-travelers-hub/rockets/SET_ROCKETS';
+const ERROR_ROCKETS = 'space-travelers-hub/rockets/ERROR_ROCKETS';
```

Comment:
Good job @alexander16108 there are a few changes to make

> If you look at the file structures we have here is not the "DUCKS" introduced to us by microverse

> Kindly put your action types in the store/reducers/types folder.

> that be find for the group to follow the file structure

Reply:
what do u mean by "DUCKS"